simpledotorg/simple-server
# 2838 assigned patients detail page

**Story card:** [ch2838](https://app.clubhouse.io/simpledotorg/story/2838/add-total-assigned-patients-to-reports-details-page)

## Because

We're adding assigned patients to the regions detail view.

## This addresses

Users' desire to see assigned patients data. 

## Test instructions

Both with and without the report_with_exclusions flag on:
- Look at reports details page for a region and confirm it matches mocks. 
- This does not apply to individual facilities
- also confirm that tables in the app look correct in general. The one line of css changed here can impact other tables. 

## Deployment note
This will require busting the cache for the `district_analytics_query.rb `.
